packages/manager/src/features/IAM/Users/UsersTable/CreateUserDrawer.tsx

```diff
@@ -113,25 +95,6 @@ export const CreateUserDrawer = (props: Props) => {
           rules={{ required: 'Email is required' }}
         />
 
-        <Controller
-          render={({ field }) => (
-            <FormControlLabel
-              label={`This user will have ${
-                field.value ? 'limited' : 'full'
-              } access to account features.
-                    This can be changed later.`}
-              onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                field.onChange(!e.target.checked);
-              }}
-              checked={!field.value}
-              control={<Toggle data-qa-create-restricted />}
-              sx={{ marginTop: 1 }}
-            />
-          )}
-          control={control}
-          name="restricted"
-        />
-
         <Box sx={{ marginTop: 1 }}>
           <Notice
             text="The user will be sent an email to set their password"
```

Comment:
```suggestion
            text="The user will be sent an email to set their password."
```

Optional, since this is existing: a period at the end of this sentence would be nice to have.